scripts/SS.py

```diff
@@ -1,58 +0,0 @@
-import numpy as np
-import scipy.optimize as opt
-import os
-import household as hh
-import firms
-import aggregates as aggr
-
-
-
-def get_SS(params, graph=False):
-    (KL_init, beta, sigma, chi_n, l_tilde, b, upsilon, S, alpha, A,
-        delta) = params
-    dist = 10
-    mindist = 1e-08
-    maxiter = 500
-    ss_iter = 0
-    xi = 0.2
-
-    r_params = (alpha, A, delta)
-    w_params = (alpha, A)
-
-    while dist > mindist and ss_iter < maxiter:
-        ss_iter += 1
-        K, L = KL_init
-        r = firms.get_r(K, L, r_params)
-        w = firms.get_w(K, L, w_params)
-        c1_guess = 0.5
-        c1_params = (r, w, beta, sigma, chi_n, l_tilde, b, upsilon, S)
-        results_c1 = opt.root(hh.get_b_sp1, c1_guess, params=(c1_params))
```

Comment:
Some of these functions are not defined in "households.py" function. I tried to check this SS with mine and household.py but have to check-in and board now. Please check them. There are some changes required to be done in the current SS and households.py
Thank you